tck-dist/src/main/asciidoc/sections/07.2ee-mode.adoc

```diff
@@ -38,6 +38,9 @@ All test classes that require running on a Jakarta profile are annotated with th
 
 When running against a Jakarta profile, include the group that matches your supported profile. For example: `<groups>core<groups>`.
 
+When running against a Jakarta profile, include the system property `<jakarta.tck.skip.deployment>false</jakarta.tck.skip.deployment>`.
```

Comment:
The false value here is a double negative, which is confusing. In order to deploy, you essentially need to ask for "don't not deploy".  This would be fine if deployment is the default, in which case no one ever needs to specify the double negative.  I think the default value should at least be reversed so that it makes more sense.